skills/record-plan/scripts/generate_acceptance.py

```
@@ -0,0 +1,85 @@
+import json
+import re
+from datetime import date
+from pathlib import Path
+
+def extract_tasks(plan_content: str) -> list[dict]:
+    """Extract tasks from markdown plan."""
+    features = []
+    task_pattern = r'^### Task \d+: (.+)$'
+
+    # Find all task headers
+    for idx, line in enumerate(plan_content.split('\n')):
```

Ruff (0.14.8)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)